agents/git-credentials/src/main/resources/org.eclipse.che.git.script.sh

```diff
@@ -34,7 +36,7 @@ echo 'api_url='"$api_url" >> ${SCRIPT_FILE}
 echo 'request="'${request}'"' >> ${SCRIPT_FILE}
 # Ssh key request may return key with decoded '=' symbol, so need to replace '\u003d' to '='.
 # TODO remove the replacement after https://github.com/eclipse/che/issues/5253 will be fixed.
-echo 'ssh_key=$(${request} "$api_url/ssh/vcs/find?name=$host$(if [ -n "$token" ]; then echo "&token=$token"; fi)"| grep -Po '\''"privateKey":.*?[^\\\\]",'\''| sed -e "s/\"privateKey\":\"//" | sed -e "s/\\\\\u003d/=/g")' >> ${SCRIPT_FILE}
+echo 'ssh_key=$(${request} "$api_url/ssh/vcs/find?name=$host$(if [ -n "$token" ]; then echo "&token=$token"; fi)"| grep -Po '\''"privateKey":.*?[^\\\\]",'\''| sed -e "s/\"privateKey\":\"//" | sed -e "s/\",//" | sed -e "s/\\\\\u003d/=/g")' >> ${SCRIPT_FILE}
```

Comment:
Remove `,`:
`sed -e "s/\"//"`

Reply:
@tolusha I couldn't reproduce the error we were having last week anymore so I've just made the PR fix the missing ssh dir for now.